cluster/addons/fluentd-elasticsearch/es-image/elasticsearch_logging_discovery.go

```diff
@@ -45,12 +46,21 @@ func main() {
 	if err != nil {
 		glog.Fatalf("Failed to make client: %v", err)
 	}
+	namespace := api.NamespaceSystem
+	envNamespace := os.Getenv("NAMESPACE")
+	if envNamespace != "" {
+		_, err = c.Namespaces().Get(envNamespace)
```

Comment:
This should be `if _, err := c.Namespaces().Get(envNamespace); err != nil { ... }`